src/components/FilterTabs/FilterTabList.tsx

```diff
@@ -0,0 +1,34 @@
+import { ButtonGroupProps, useStyleConfig, ButtonGroup } from "@chakra-ui/react"
+import { FC } from "react"
+import { FilterTabVariant, FilterTabSize } from "../../theme/FilterTab"
+import { FilterTabListProvider } from "./context"
+
+export interface FilterTabListProps extends ButtonGroupProps {
```

Comment:
Maybe just `FilterTbas`? I think `List` is unnecessary. If you agree let's get rid of `List` in other places.

Reply:
Fixed in af7585f97f9f527a265a92441fd248f215044f86